charts/homebridge/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+appVersion: 3.1.0
+version: 1.0.0
+name: homebridge
```

Comment:
It's a minor thing, but can you include an icon for this chart as well?  It makes it looks a lot nicer when browsing a place like [artifacthub](https://artifacthub.io/packages/search?page=1&org=k8s-at-home)